app/javascript/components/public_health/PatientsTable.js

```diff
@@ -93,6 +98,33 @@ class PatientsTable extends React.Component {
     // select tab and fetch patients
     this.handleTabSelect(tab);
 
+    // Select page if it exists in local storage
+    let page = localStorage.getItem(`${this.props.workflow}Page`);
```

Comment:
I think the tricky thing about saving the page number is that the tables are frequently changing and let's say a user is saved on page 5 or something of the asymptomatic tab, but the next day there aren't 5 pages of asymptomatic monitorees because they didn't report or they were symptomatic, so we might run into some unexpected behavior with this one.

Reply:
Agreed, I made this a bit smarter.